packages/aws/aws.0.1.0/url

```diff
@@ -0,0 +1,2 @@
+url:"https://github.com/inhabitedtype/ocaml-aws/releases/download/aws-v0.1.0/aws-v0.1.0.tar.gz"
```

Comment:
This should be:

`archive: "https://github.com/inhabitedtype/ocaml-aws/releases/download/aws-v0.1.0/aws-v0.1.0.tar.gz"`